src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -919,6 +919,28 @@
     /**
      * The {@code -} (unary minus) operator.
      *
+     * <p>For example:</p>
+     *
+     *  <pre>
+     *   a=-b;
+     * </pre>
+     *
+     * <p>parses as:</p>
+     *
+     * <pre>
+     *   +--UNARY_MINUS
+     *       |
+     *       +--EXPR
+     *          |
+     *          +--ASSIGN (=)
+     *              |
+     *              +--IDENT (a)
+     *              +--MINUS (-)
+     *                  |
+     *                  +--IDENT (b)
```

Comment:
@gurleennsidhuu Please share CLI output, and site output. Also, please match CLI output with your changes. These changes doesn't seem correct to me.